docs/index.md

```diff
@@ -4,6 +4,14 @@ title: Home
 
 # ![Ragna logo](assets/brand/logo-lockup-horizontal/logo-lockup-horizontal.png)
 
+- Here is a nested list with a tab width of 2
+  - First item
+  - Second item
+- Here is a nested list with a tab width of 4
+    - First item
+    - Second item
```

Comment:
Lint CI is also triggering now, but this format with the extension gives this result

![image](https://github.com/Quansight/ragna/assets/6849766/f1712022-6205-4ab7-92b5-f0c8af16dd86)


Since we don't want to use a tab width of 4, this is fine. Of course we need to revert 06a71a8ed7cb24135814b42b5aee5fcba56c56af before merge.